### ISSUE TITLE
Android build error in react native v0.69.0

### ISSUE THREAD

**Issue by jabirmhd** [daily-co/react-native-daily-js#27]
I was upgrading my app from RN 0.67.3 to 0.69.0, after the completing the upgrade I am not able to build the app for android
Following is the error I am getting.

```
* Where:
Build file '<project_dir>/node_modules/@daily-co/react-native-daily-js/android/build.gradle' line: 115

* What went wrong:
A problem occurred configuring project ':daily-co_react-native-daily-js'.
> Could not resolve all files for configuration ':daily-co_react-native-daily-js:implementation'.
   > Could not resolve com.facebook.react:react-native:+.
     Required by:
         project :daily-co_react-native-daily-js
      > Cannot choose between the following variants of com.facebook.react:react-native:0.69.0:
          - debugVariantDefaultRuntimePublication
          - releaseVariantDefaultRuntimePublication
        All of them match the consumer attributes:
          - Variant 'debugVariantDefaultRuntimePublication' capability com.facebook.react:react-native:0.69.0:
              - Unmatched attributes:
                  - Provides com.android.build.api.attributes.BuildTypeAttr 'debug' but the consumer didn't ask for it
                  - Provides org.gradle.category 'library' but the consumer didn't ask for it
                  - Provides org.gradle.dependency.bundling 'external' but the consumer didn't ask for it
                  - Provides org.gradle.libraryelements 'aar' but the consumer didn't ask for it
                  - Provides org.gradle.status 'release' but the consumer didn't ask for it
                  - Provides org.gradle.usage 'java-runtime' but the consumer didn't ask for it
          - Variant 'releaseVariantDefaultRuntimePublication' capability com.facebook.react:react-native:0.69.0:
              - Unmatched attributes:
                  - Provides com.android.build.api.attributes.BuildTypeAttr 'release' but the consumer didn't ask for it
                  - Provides org.gradle.category 'library' but the consumer didn't ask for it
                  - Provides org.gradle.dependency.bundling 'external' but the consumer didn't ask for it
                  - Provides org.gradle.libraryelements 'aar' but the consumer didn't ask for it
                  - Provides org.gradle.status 'release' but the consumer didn't ask for it
                  - Provides org.gradle.usage 'java-runtime' but the consumer didn't ask for it
```

# Steps to reproduce

Rebuild the app after upgrading to 0.69.0

# System information 

 

```
 "@daily-co/react-native-daily-js": "^0.25.0",
 "@daily-co/react-native-webrtc": "^1.94.1-daily.4",
```

   OS: macOS 12.4
    CPU: (8) arm64 Apple M1
    Memory: 130.44 MB / 8.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.14.0 - ~/.nvm/versions/node/v16.14.0/bin/node
    Yarn: 1.22.17 - ~/.nvm/versions/node/v16.14.0/bin/yarn
    npm: 8.3.1 - ~/.nvm/versions/node/v16.14.0/bin/npm
    Watchman: 2022.05.16.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /opt/homebrew/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.1 AI-211.7628.21.2111.8193401
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.11 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.0.0 => 18.0.0 
    react-native: 0.69.0 => 0.69.0 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

**Comment by filipi87** [comment]
Just to give you some feedback, we have already fixed It and have an open PR. It will probably be merged before our next release.

**Comment by filipi87** [comment]
Hi, just to give you an update, the fix has already been merged and will be available on the next release, 0.26.0.

**Comment by jabirmhd** [comment]
> Hi, just to give you an update, the fix has already been merged and will be available on the next release, 0.26.0.

@filipi87 I can confirm that this issue is still not resolved in version 0.26.0.

**Comment by filipi87** [comment]
Hi you are right, we had already created the release branch 0.26 before these changes were merged, so unfortunately those changes will be available on the version 0.27.0, which we intend to release next week. Sorry for the delay.

**Comment by filipi87** [comment]
The fix is available on version `0.27.0`.
